src/data/systemIntake.ts

```diff
@@ -1,4 +1,44 @@
-import { SystemIntakeForm } from 'types/systemIntake';
+import {
+  GovernanceCollaborationTeam,
+  SystemIntakeForm
+} from 'types/systemIntake';
+import cmsGovernanceTeams from '../constants/enums/cmsGovernanceTeams';
+
+export const initialSystemIntakeForm: SystemIntakeForm = {
+  id: '',
+  euaUserID: '',
+  projectName: '',
+  status: 'DRAFT',
+  requester: {
+    name: '',
+    component: ''
+  },
+  businessOwner: {
+    name: '',
+    component: ''
+  },
+  productManager: {
+    name: '',
+    component: ''
+  },
+  isso: {
+    isPresent: null,
+    name: ''
+  },
+  governanceTeams: {
+    isPresent: false,
```

Comment:
`isPresent` should be `null` or else the question has `None of the above` preselected.